Azure-Samples/chat-with-your-data-solution-accelerator
# refactor: Apply pep8 on leftover files from pr-merge

## Purpose

* apply pep8 on leftover files from pr-merge
* Resolves #840

## Does this introduce a breaking change?



## How to Test
*  Get the code
* Run the tests:
```
poetry run pytest -m "not azure and not functional"    
poetry run pytest -m "functional"  

```
* Run the application locally and go to http://localhost:8502/Ingest_Data
* Verify all options work
